ores/lock_manager/poolcounter.py

```diff
@@ -0,0 +1,54 @@
+"""
+Class to implement PoolCounter lock manager.
+
+This is useful to make sure too many connections are not coming from one IP.
+"""
+
+import socket
+
+from .lock_manager import LockManager
+
+
+class PoolCounter(LockManager):
+    def __init__(self, config):
+        self.server = config['server']
+        self.port = config['port']
+        self.stream = None
+
+    def connect(self):
+        self.stream = socket.socket(socket.AF_INET, socket.SOCK_STREAM, 0)
+        return self.stream.connect((self.server, self.port))
+
+    def lock(self, key, workers, maxqueue, timeout):
+        if not self.stream:
+            self.connect()
+        try:
+            self.stream.send(bytes('ACQ4ME %s %d %d %d\n' % (key, workers, maxqueue, timeout), 'utf-8'))
+            data = self.stream.recv(4096).decode('utf-8')
+            print(data)
+        except socket.error as e:
+            self.stream = None
+            raise e
+
+        return data.strip() == 'LOCKED'
+
+    def release(self, key):
+        if not self.stream:
+            return False
+
+        try:
+            self.stream.send(bytes('RELEASE %s\n' % key, 'utf-8'))
```

Comment:
Normally this would be `format` in py 3, but now I've typed more characters than you saved ;-)